packages/@sanity/form-builder/src/inputs/Array/ItemValue.js

```diff
@@ -118,19 +119,32 @@ export default class RenderItemValue extends React.Component<Props> {
     this._focusArea = el
   }
 
+  handleDialogAction = action => {
+    if (!action) {
+      console.error('no action')
```

Comment:
Is this intentional?

Reply:
Yes. I think it would be nice for a developer to get that error in the console.